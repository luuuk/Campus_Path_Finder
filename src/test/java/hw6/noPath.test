# This test tests the case of no path between two nodes

LoadGraph test test.tsv

FindPath test a d