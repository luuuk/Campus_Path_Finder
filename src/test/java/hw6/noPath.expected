# This test tests the case of no path between two nodes

loaded graph test

path from a to d:
no path found